src/js/media/controllers/library.js

```diff
@@ -196,7 +196,7 @@ Library = wp.media.controller.State.extend(/** @lends wp.media.controller.Librar
 	isImageAttachment: function( attachment ) {
 		// If uploading, we know the filename but not the mime type.
 		if ( attachment.get('uploading') ) {
-			return /\.(jpe?g|png|gif|webp|avif)$/i.test( attachment.get('filename') );
+			return /\.(jpe?g|png|gif|webp|avif|heif)$/i.test( attachment.get('filename') );
```

Comment:
The file extension for HEIF images is typically `.heic`, but there's also `.heics`,`.heif` and `.heifs`. The "s" is for sequences.

Reply:
ah right, for this PR I'm trying to add just support for `.heic` files which I believe is what ip[hones use as their default output format. Do you think there is value is supporting all of the possible extensions?

Reply:
I don't think so, haven't seen those in the wild yet.